.circleci/config.yml

```diff
@@ -168,6 +168,9 @@ jobs:
             if [[ -n "$DOCKER_PASS" ]]; then
                   docker login -u $DOCKER_USER -p $DOCKER_PASS
                   docker push ${CONTAINER_NAME}:${DOCKER_TAG}
+                  echo "Tagging latest image..."
+                  docker tag ${CONTAINER_NAME}:${DOCKER_TAG} ${CONTAINER_NAME}:latest
+                  docker push ${CONTAINER_NAME}:latest
```

Comment:
The purist in me is a bit hesitant to make a "latest" tag standard. It gives people a way to not explicitly define which tag/version they want. Which means it becomes a nightmare to try and work out what they actually used later on.

It is super convenient to have latest, but for your long term health it isn't so smart. A bit like eating only cake for breakfast is super cool :-/

Reply:
It's not standard, but it's provided because people want and expect it. if I had a dollar for every time someone posted an issue because they claimed the endpoint didn't work (and had just misspelled the tag). I'm all for encouraging reproducibility, but making it really hard to use a tool I don't think is the right way to do it.

Reply:
¯\_(ツ)_/¯ you decide

Reply:
thank you!